inspire/modules/authors/recordext/models/author.cfg

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+## This file is part of INSPIRE.
+## Copyright (C) 2014, 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+bases:
+    author_base
+fields:
+    ids
+    name
+    publications_list
+    emails
+    positions
+    other_names
+    previous_names
+    private
+    field_categories
+    curators_note
+    source
+    prizes
+    public_note
+    experiments
+    phd_advisors
+    conferences
+    urls
+    native_name
+    date_added
+    date_updated
```

Comment:
To make things clearer, could you group more or less together all semantically related fields? (Here and in author.cfg) E.g. all name fields together.